### ISSUE TITLE
Icon Requests

### ISSUE THREAD

**Issue by tdoerrich** [dkanada/frost#488]
Hello! Here are some apps I believe is missing.

- Aurora Store (just broken?)
- Budget My Life
- Kinder und Gift
- Microsoft Launcher
- DHL (broken?)
- RunnerUp
- Datura Firewall
- Recorder
- Organic Maps

I believe I added all from the .zip. Though I could have missed one or two from the zip file.

[20210630_053847.zip](https://github.com/dkanada/frost/files/6737839/20210630_053847.zip)

**Comment by baitmooth** [comment]
-- | -- | -- | -- | --
-- | -- | -- | -- | --
budgetmylife (*n*) | firewall (*n*) | kinderundgift (*n*) | organicmaps (*n*) | recorder (*u*) 

![488-2021-06-30-1338](https://user-images.githubusercontent.com/23211478/123957022-9fe2ad80-d9ab-11eb-85f8-df22c3cc19c8.png)

- [x] Aurora Store
- [x] Budget My Life
- [x] Kinder und Gift
- [ ] Microsoft Launcher *)
- [x] DHL
- [x] RunnerUp
- [x] Datura Firewall
- [x] Recorder
- [x] OpenCV Manager
- [x] Organic Maps

*) should work. I did not see why it does not, but will check again later.

**Comment by baitmooth** [comment]
I really don't see why 'Microsoft Launcher does not show up for you. Here is what you requested:
`
<item component="ComponentInfo{com.microsoft.launcher/com.microsoft.launcher.Launcher}" drawable="microsoftlauncher"/>`
And whats impleneted:
`
 <item component="ComponentInfo{com.microsoft.launcher/com.microsoft.launcher.Launcher}" drawable="microsoftlauncher" />`

Do you have an up to date version of frost?

**Comment by tdoerrich** [comment]
Yeah, I have the latest version. I'll wait for the next update and see. Maybe that fixes it.
